ISM43362/ISM43362.h

```diff
@@ -248,6 +254,34 @@ class ISM43362 {
     // Connection state reporting
     nsapi_connection_status_t _conn_status;
     mbed::Callback<void()> _conn_stat_cb;
+
+    typedef struct
+    {
+      char cc[5];
```

Comment:
Why is this 5 bytes long ? most codes are 2 bytes and END is 3 BYTES, with 0 ending char isn't 4 bytes enough ? 
Could even be 3 bytes by using a 2 bytes code as last array entry ...